morphir/runtime/src/org/finos/morphir/runtime/quick/EvaluatorQuick.scala

```diff
@@ -23,7 +23,7 @@ object EvaluatorQuick {
   type FloatType = Double
 
   private[runtime] def evalAction(
-      value: Value[TypeAttribs, ValueAttribs],
+      value: Value[scala.Unit, UType],
```

Comment:
This case is more complicated: These did refer to `TypedMorphirRuntimeDefs`, but in a weird way where the `Value[_, _]` type was manually constructed. I think this should just be `value : TypedValue` - @DamianReeves , do you agree?

Reply:
Yeah I think that is fine for this one.